src/TBuffer.cpp

```diff
@@ -2532,8 +2539,12 @@ void TBuffer::decodeSGR(const QString& sequence)
                 case 24:
                     mUnderline = false;
                     break;
-                // case 25:
-                //    break; // blink off
+                 case 25:
+                     if (!mItalicBeforeBlink) {
+                         mItalics = false;
+                     }
+                     mItalicBeforeBlink = true;
```

Comment:
shouldn't this be false, to match the default? I may be missing something but this seems like after the first time blink was turned off it would just leave mItalicBeforeBlink set to true for the TBuffer, with nothing to set it back to false.

Reply:
Definitely, I meant for this to be reset.

Reply:
Fixed.